index.js

```diff
@@ -3,6 +3,10 @@
 
 var index = require('./package.json');
 
+function isFunction(functionToCheck) {
+	return functionToCheck && {}.toString.call(functionToCheck) === '[object Function]';
```

Comment:
It looks like this came from https://stackoverflow.com/a/7356528/796832

Seems like we should go with the more concise and straightforward answer, https://stackoverflow.com/a/6000009/796832

```
return typeof functionToCheck === "function"
```

Reply:
The straightforward answer looks more concise indeed, I tested it and it seems to work so I switched to it thanks!